src/cmdstan/stansummary_helper.hpp

```diff
@@ -301,11 +300,9 @@ Eigen::VectorXd percentiles_to_probs(
         throw std::exception();
       cur_pct = pct;
     } catch (const std::exception &e) {
-      std::stringstream message_stream("");
-      message_stream << "position " << i << ", value " << percentiles[i] << ". "
-                     << "Values must be in range (1,99), inclusive, "
-                     << "and strictly increasing.";
-      throw boost::program_options::error(message_stream.str());
+      throw std::invalid_argument(
+          "values must be in range (1,99)"
+          ", inclusive, and strictly increasing.");
```

Comment:
I think there needs to be a newline after the error. Dunno where to add this to the code, but when I hit the error my cursor ends up way over at the right:

```
bbales2@tadpole:~/cmdstan-develop$ ./bin/stansummary --percentiles=-10,3,4 output2.csv
Option --percentiles -10,3,4: values must be in range (1,99), inclusive, and strictly increasing.bbales2@tadpole:~/cmdstan-develop$ 
```

Reply:
leaving this message the same, added newline at point where error is reported - L 89 stansummary.cpp